app/src/main/java/io/appium/settings/LocationService.java

```diff
@@ -77,9 +78,17 @@ public int onStartCommand(Intent intent, int flags, int startId) {
                 return START_NOT_STICKY;
             }
         }
+
+        // https://stackoverflow.com/a/45047542
+        // https://developer.android.com/about/versions/oreo/android-8.0-changes.html
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            Log.w(TAG, "Starting location service");
```

Comment:
I'd rather change the logging level to info or debug

Reply:
👍  added it to check the log, forgot to revert it. Changing it to info.